src/core/json/jsonpath_test.cc

```diff
@@ -123,6 +129,19 @@ TEST_F(JsonPathTest, Parser) {
   EXPECT_EQ(0, Parse("$.plays[*].game"));
 }
 
+TEST_F(JsonPathTest, Root) {
```

Comment:
the only change is this test. Now for empty Path we return the whole object to satisfy `$` query.